### ISSUE TITLE
Early stopping feature

### ISSUE THREAD

**Issue by nupoor-ka** [roboflow/multimodal-maestro#198]
### Search before asking

- [x] I have searched the Multimodal Maestro [issues](https://github.com/roboflow/multimodal-maestro/issues) and found no similar feature requests.

### Question

Hi, I've been using Maestro to finetune Florence2 on a custom dataset, it's a great tool. One issue I've faced is that I have to guess at the number of epochs I should run fine-tuning for instead of allowing for an early-stop. Is there any way to set it up to stop at a certain minimum loss value or stop when the change in training loss reaches a certain limit?
